scipy/special/_precompute/wright_bessel.py

```diff
@@ -265,7 +265,7 @@ def fp(eps, a, b, x, phi):
     def arclength(eps, a, b, x, epsrel=1e-2, limit=100):
         """Compute Arc length of f.
 
-        Note that the arg length of a function f fro t0 to t1 is given by
+        Note that the arg length of a function f from t0 to t1 is given by
```

Comment:
There is another typo here: `arg` -> `arc`

Reply:
This one is difficult to catch without processing n-grams instead of isolated words.